src/mm2_audio.h

```diff
@@ -153,7 +153,7 @@ namespace MM2
     class mmDirSnd : public DirSnd {
     protected:
         UINT eaxEnabled;
-        UINT dsound3DEnabled;
+        float volume;
```

Comment:
Are you absolutely sure about this? I don't see any references to a float value being in `mmDirSnd`, and this one is returned from `mmDirSnd::DSound3DEnabled` as a uint. Not a float.